readme.md

```diff
@@ -22,8 +23,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://DenScriptor.github.io/layout_stars/)
+- [TEST REPORT LINK](https://DenScriptor.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="934" alt="image" src="https://github.com/user-attachments/assets/b8f92cc1-7786-4511-bad5-60f2e7e1dea2" />

Reply:
feel free to ask for some help in the chat how to fix them